apps/bug-cd/src/app/main-navigation.component.ts

```diff
@@ -34,7 +35,7 @@ interface MenuItem {
   },
 })
 export class NavigationComponent {
-  @Input() menus!: MenuItem[];
+  @Input() menus: MenuItem[] | undefined;
```

Comment:
We really want to have menu items here. 

We should write `@Input({required: true}) menu !: MenuItem[]`

But we can pass an empty array if no item in the menu

Reply:
You are right, needed for the type emitting, but figured out another approach